server.js

```diff
@@ -25,10 +47,69 @@ app.use(cors());
 app.use(express.json());
 
 // Start defining your routes here
+
+//Start page 
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send({
+  Message: "Avocado sale data", 
+  Routes: [{ 
+    "/avocadoSales": "all of the data",
+    "/avocadoSales/:date": "example: /avocadoSales/2015-10-25",
+    "/highestPrice": "Sorts the data in order of highest price",
+    "/avocados/:id": "example: /avocadoSales/id/63834553a60461fc8065b2bf"
+  }]
+ 
+  });
```

Comment:
This is also very smart (i did the same this week ;) ) this makes it easier for whoever to know what to search for 👍